ElectronClient/app/gui/MainScreen.jsx

```diff
@@ -132,16 +132,19 @@ class MainScreenComponent extends React.Component {
 			});
 		} else if (command.name === 'setTags') {
 			const tags = await Tag.tagsByNoteId(command.noteId);
-			const tagTitles = tags.map((a) => { return a.title }).sort();
+			const tagObjects = tags.map((a) => { return {value: a.id, label: a.title }}).sort();
```

Comment:
I think we need a custom sort function here to sort by title. Calling `sort()` on objects gives probably random results.

Reply:
And could you rename it to `noteTags` please? That way it makes it clear that it's only the current tags of the notes.

Reply:
You are absolutely right about sort, that was an oversight. It clearly wasn't sorting.